studio/runner.py

```diff
@@ -268,17 +268,39 @@ def main(args=sys.argv[1:]):
     logger.debug('resources requested: ')
     logger.debug(str(resources_needed))
 
-    artifacts = dict()
-
     # Set up default artifacts:
-    workspace_art = {
-        'mutable': False,
-        'local': os.getcwd(),
-        'unpack': True
+    artifacts = {
+        'workspace': {
+            'mutable': False,
+            'local': os.getcwd(),
+            'unpack': True
+        },
+        'modeldir': {
+            'mutable': True,
+            'unpack': True
+        },
+        'retval': {
+            'mutable': True,
+            'unpack': True
+        },
+        'output': {
+            'mutable': True,
+            'unpack': True
+        },
+        'tb': {
+            'mutable': True,
+            'unpack': True
+        },
+        '_metrics': {
+            'mutable': True,
+            'unpack': True
+        },
+        '_metadata': {
+            'mutable': True,
+            'unpack': True
+        }
```

Comment:
As before, file paths for local mutable artifacts are populated during Experiment object creation,
this is when we know experiment ID + local artifact path (which will depend on the client/server side currently running).